File: official/docs/csharp/current/billing/add-stripe-credit-card.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            EasyPost.Models.API.PaymentMethod paymentMethod = await Client.ReferralCustomer.AddCreditCardFromStripe(
                "REFERRAL_USER_API_KEY",
                "seti_...",
                EasyPost.Models.API.PaymentMethod.Priority.Primary
            );

            Console.WriteLine(JsonConvert.SerializeObject(paymentMethod, Formatting.Indented));
        }
    }
}

File: official/docs/node/current/billing/add-stripe-credit-card.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const paymentMethod = await client.ReferralCustomer.addCreditCardFromStripe(
    'REFERRAL_USER_API_KEY',
    'seti_...',
    'primary',
  );

  console.log(paymentMethod);
})();
